crates/vm/levm/src/opcode_handlers/stack_memory_storage_flow.rs

```diff
@@ -329,11 +330,45 @@ impl VM {
         self.increase_consumed_gas(current_call_frame, gas_cost::JUMP)?;
 
         let jump_address = current_call_frame.stack.pop()?;
-        current_call_frame.jump(jump_address)?;
+        Self::jump(current_call_frame, jump_address)?;
 
         Ok(OpcodeSuccess::Continue)
     }
 
+    /// JUMP* family (`JUMP` and `JUMP` ATTOW [DEC 2024]) helper
+    /// function.
+    /// This function returns whether the `jump_address` is a valid JUMPDEST
+    /// for the specified `call_frame` or not.
+    fn is_valid_jump_addr(call_frame: &CallFrame, address: usize) -> bool {
```

Comment:
```suggestion
    fn is_valid_jump_addr(call_frame: &CallFrame, jump_address: usize) -> bool {
```
Last thing, I promise

Reply:
Hahha, make as many suggestions as you see fit Jere!!
No worries.

I'll change it ASAP

Reply:
Done! 😊